serverless/email/webhook/input.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/google/uuid"
+)
+
+/*
+	Supported resource types:
+		DEPOSIT_CRYPTO
+		SUPPORT_TICKET
+		KYC_LEVEL
+		DEPOSIT_FIAT
+		ACCOUNT_FROZEN
+*/
+
+type emailPayload struct {
+	Email        string                 `json:"email"`
```

Comment:
Where are we validating `email` and all the other fields to be correctly formatted?

Reply:
addressed here: https://github.com/brave-intl/bat-go/pull/1593/commits/70265af51d6318fdd53c2d07899ff7b5bb969512